### DISCUSSION TITLE
Using limma on Pearson residuals from SCTransform?

### DISCUSSION THREAD

**Discussion by kunleng** [satijalab/seurat#4569] [Q&A] [answered]
Hi all, I am an avid Seurat user from the Kampmann lab at UCSF. We have been doing some pooled CRISPR single-cell experiments which can have complex experimental designs. Doing t-test on Pearson residuals from SCTransform has worked well for simple comparisons, but I'm wondering if it would be appropriate to use limma on the Pearson residuals to accommodate more complex designs? i.e. interaction terms, possibly mixed-effects modeling

**Comment by saketkc** [comment, marked as answer]
Hi @kunleng, yes, you should be able to to plugin residuals in a linear modeling framework with complex design as long as the residuals come from the same SCT model.

**Reply by kunleng** [reply]
Thanks so much Saket!
